src/components/InputComponent.tsx

```diff
@@ -0,0 +1,13 @@
+type InputProps = {
+  className: string;
+  text: string;
+};
+
+export const InputComponent = (props: InputProps) => {
+  return (
+    <div className={props.className}>
+      <div>{props.text}</div>
+      <input type="type" />
```

Comment:
please update this component
there is no input with type="type"
looks like you wanted to use it from props
```tsx
export const InputComponent = (props: InputProps) => {
  return (
    <div className={props.className}>
      <div>{props.text}</div>
      <input type={props.type} />
```